docs/real-world-examples/ccfraud.md

```diff
@@ -31,6 +31,30 @@ In the next section, we will run a job in the AzureML workspace that will unpack
 
 Kaggle required username and a developer key, so we will first store safely those credentials in the workspace keyvault.
 
+### Using Azure CLI
```

Comment:
Can you change title "Option 1 : using Azure CLI" and same for Option 2 ? so people understand those are 2 different paths